packages/hoppscotch-app/src/components/environments/my/index.vue

```diff
@@ -122,4 +126,15 @@ const editEnvironment = (environmentIndex: number | "Global") => {
 const resetSelectedData = () => {
   editingEnvironmentIndex.value = null
 }
+
+defineActionHandler("modals.my.environment.edit", (args?: any[]) => {
+  const [envName, variableName]: any = args
+  editingVariableName.value = variableName
+  const envIndex: number = environments.value.findIndex(
+    (environment: Environment) => {
+      return environment.name === envName
+    }
+  )
+  editEnvironment(envIndex >= 0 ? envIndex : envName)
```

Comment:
Since you have defined `envName` as `any`. You have to provide guarantee that `envName` is either a number or `"Global"` to satisfy the type contract.
Please make sure to add the assertion that `envName` is `"Global"` before calling `editEnvironment` to make sure you keep its contract and not bring possibility of UB.